workflows/get_species_status.py

```diff
@@ -153,27 +183,44 @@ def get_species_synonyms(species_status: dict) -> None:
             sp["moreSynonyms"] = True if len(data) == 50 else False
 
 
-def get_vernaculars(species_status: dict) -> None:
+def get_vernaculars(species_status: dict, URI: str, params: dict) -> None:
     for i, sp in enumerate(species_status.values()):
         if sp:
             print(
                 f"Getting Species number {i + 1}'s common names. \
-                    ID:{sp['valid_AphiaID']}"
+                    ID:{sp[params['valid_id']]}"
             )
-            data = requests.get(vernacularsByAphiaID.format(sp["valid_AphiaID"]))
+            data = requests.get(URI.format(sp[params['valid_id']]))
 
             sp["commonNames"] = data.json() if data.status_code == 200 else []
 
 
 if __name__ == "__main__":
-    species_status = get_species_status(index_species, LIMIT)
+    working_on: str = "irmng"
```

Comment:
My understanding of this part is that the current script only works on missing stuff and not the WORMS data, correct? Can you add a CLI flag to determine which part the script should work on?